docs/src/language_servers.md

```diff
@@ -453,7 +197,6 @@ Follow installation instructions on [LSP-json](https://github.com/sublimelsp/LSP
 
 3. Open `Preferences > Package Settings > LSP > Settings` and add the `"julials"` client configuration to the `"clients"`:
 
-=== "Sublime Text 4"
     ```json
     {
         "clients": {
```

Comment:
Re-indent to the left each of these lines?

Reply:
I was under the assumption that this indent was needed so that it is part of the ordered list. Is this not needed?